src/site/layouts/vamc_operating_status_and_alerts.drupal.liquid

```diff
@@ -71,7 +71,7 @@
                         </ul>
                     </section>
 
-                    {% if fieldBannerAlert.0.entity and fieldBannerAlert.0.entity.status == true %}
```

Comment:
why lose the check for published?

Reply:
That's already being handled by graphql, and this causes alerts that comes after the first one to not output, even if they are published, @swirtSJW